downstream/modules/platform/con-aap-upgrade-prereq.adoc

```diff
@@ -0,0 +1,147 @@
+[id="con-aap-upgrade-prereq_{context}"]
+
+= Prerequisites for upgrading {PlatformNameShort}
+
+== Node requirements
+
+The following specifications are required for the nodes involved in the {PlatformNameShort} upgrade process:
+
+* 16 GB of RAM for controller nodes, database node, execution nodes and hop nodes.
+* 4 CPUs for controller nodes, database nodes, execution nodes, and hop nodes.
+* 150 GB+ disk space for database node.
+* 40 GB+ disk space for non-database nodes.
+* DHCP reservations use infinite leases to deploy the cluster with static IP addresses.
+* DNS records for all nodes.
+* Red Hat Enterprise Linux 8 or later 64-bit (x86) installed for all nodes.
+* Chrony configured for all nodes.
+
+== {ControllerNameStart} configuration requirements
+
+.Configuring NTP server using Chrony
+
+Each {PlatformNameShort} node in the cluster must have access to an NTP server. Use the `chronyd` to synchronize the system clock with NTP servers. This ensures that when cluster nodes use SSL certificates that require validation, they do not fail if the date and time between the nodes are not in sync.
+
+This is required for all nodes used in the upgraded {PlatformNameShort} cluster:
+
+. Install `chrony`:
++
+----
+# dnf install chrony --assumeyes
+----
+. Open `/etc/chrony.conf` using a text editor.
+. Locate the public server pool section and modify it to include the appropriate NTP server addresses. Only one server is required, but three is recommended. The `iburst` option is added to speed up the time that it takes to properly sync with the servers:
++
+----
+# Use public servers from the pool.ntp.org project.
+# Please consider joining the pool (http://www.pool.ntp.org/join.html).
+server <ntp-server-address> iburst
+----
+. Save changes within the `/etc/chrony.conf` file.
+. Start the host and enable the `chronyd` daemon:
++
+----
+# systemctl --now enable chronyd.service
+----
+. Verify the `chronyd` daemon status:
++
+----
+# systemctl status chronyd.service
+----
+
+.Attaching Red Hat subscription on all nodes
+{PlatformName} requires that you have valid subscriptions attached to all nodes. You can verify if your current node has a Red Hat subscription by running the following command:
+----
+# subscription-manager list --consumed
+----
+If there is not a Red Hat subscription attached to the node, see the https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/red_hat_ansible_automation_platform_installation_guide/planning-installation#proc-attaching-subscriptions_planning[attaching your {PlatformNameShort} subscription] guide on how to do so.
```

Comment:
Suggest changing: "guide on how to do so" to "for more information".  also remove the from "see the" so it becomes "..., see attaching your {PlatformNameShort} subscription for more information."